favorites.py

```diff
@@ -277,6 +288,48 @@ def add_group(self, s):
 
         self.obj.files["groups"][s] = []
 
+    def prompt_for_alias(self, index, name, group_name=None):
+        """Prompt for an alias for the favorite file.
+
+            Args passed by SelectFavoriteFileCommand:
+
+            index      : the evaluated quick panel index
+            name       : a tuple (alias, path)
+            group_name : the group browsed in the quick panel
+        """
+        def find_index_in(node):
+            '''Find the index of the file that has been just added.'''
+            for f in node:
+                if f['file'] == name:
+                    return index, (f['alias'], name)

```

Comment:
When a file is added, it alias defaults to basename, but it's still there, therefore I'm returning it too, it's the default alias that will be edited